app/assets/stylesheets/app/sortable.scss

```diff
@@ -1,19 +1,19 @@
 .sortable {
-  background-image: image-url("icon-sort.gif");
+  background-image: image-url("icon-sort.gif"); #
```

Comment:
@jhanggi not sure what to do about these--I'm pretty sure they depend on font-awesome, but I don't know how this `.gif`thing works or whether I can do it with the new version. Any suggestions?